### PR TITLE
Hangman: Improve hangman terms

### PR DESCRIPTION
**Pull request by Distrib-ps**

### CODE REVIEW

File: server/chat-plugins/hangman.ts [hunk 1]
```diff
@@ -428,24 +429,34 @@ export const commands: ChatCommands = {
...
 			buf += `<div class="infobox">`;
-			buf += `<strong>${t}</strong>:<br />`;
-			buf += roomTerms[t].map(
-				hint => `<button class="button" name="send" value="/hangman rr ${t},${hint},room:${room.roomid}">${hint}</button>`
-			);
+			buf += `<h3>${t}</h3><hr />`;
+			if (user.can('mute', null, room, 'hangman addrandom')) {
+				buf += roomTerms[t].map(
+					hint => `<button class="button" name="send" value="/hangman rr ${t},${hint},room:${room.roomid}">${hint}</button>`
+				);
```
Comment (Zarel) [inline server/chat-plugins/hangman.ts:453]: It seems kind of bad for clicking on the hint to delete it. Especially if there's another delete button that deletes all the hints.

Comment (Zarel) [reply server/chat-plugins/hangman.ts:453]: I guess this PR doesn't make that problem _worse_, though.

Comment (Distrib-ps) [reply server/chat-plugins/hangman.ts:453]: > It seems kind of bad for clicking on the hint to delete it. Especially if there's another delete button that deletes all the hints.

Should this option be removed?

Comment (Zarel) [reply server/chat-plugins/hangman.ts:453]: Plausibly. I'll look into it.

**Pull request merged**